components/sidebar/sidebar_channel/sidebar_channel_link/sidebar_channel_link.tsx

```diff
@@ -161,7 +169,19 @@ export default class SidebarChannelLink extends React.PureComponent<Props, State
     showChannelAsUnread = (): boolean => this.props.unreadMentions > 0 || (this.props.unreadMsgs > 0 && this.props.showUnreadForMsgs);
 
     render(): JSX.Element {
-        const {link, label, channel, unreadMentions, icon, isMuted, isChannelSelected} = this.props;
+        const {link, label, channel, unreadMentions, icon, isMuted, isChannelSelected, showTutorialTip, actions} = this.props;
+
+        let tutorialTip: JSX.Element | null = null;
+        if (showTutorialTip && channel.name === Constants.DEFAULT_CHANNEL) {
+            tutorialTip = (
+                <SidebarTutorialTip
+                    townSquareDisplayName={this.props.townSquareDisplayName}
+                    offTopicDisplayName={this.props.offTopicDisplayName}
+                    openLhs={actions.openLhs}
+                />
+
+            );
+        }
```

Comment:
Why did we bring in the `SidebarTutorialTip` here? We still have it https://github.com/mattermost/mattermost-webapp/blob/0eab33648da396ad591498bbf28d6b048bd47f1a/components/legacy_sidebar/sidebar_channel/sidebar_channel.jsx#L264-L268 there?

Reply:
It's present in the legacy sidebar and not the new sidebar